packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -154,6 +151,19 @@ export const BASIC_FALSY_TESTS = {
 
 export class SharedConditionalsTest extends AbstractConditionalsTest {
```

Comment:
I see. Turns out I wrote that code and I didn't even remember/understand it :P `AbstractConditionalsTest` is like an "interface" for the most part and `SharedConditionalsTest` has the test case implementations. Presumably, I did that for clearity but I'm not sure if it end up helping :P

But in any case, I think this supports my comment below that the `renderValues` implementation should be pushed one level down in the class hierarchy.